app/containers/Menu.js

```diff
@@ -0,0 +1,649 @@
+var React = require('react');
+var styles = require('../styles');
+var ReactRouter = require('react-router');
+var Link = ReactRouter.Link;
+
+var Menu = React.createClass({
+
+  componentWillMount: function () {
+    this.lastSort = 'Alphabetical';
+    this.setState({
+      sort: "Popularity",
+      popularButtonDisabled: true,
+      alphabeticalButtonDisabled: false,
+      drinkList: [
```

Comment:
do just want to point out this is a largee list, not desirable. Going forward, we will help you with an API for this. It's not hard at all to build a backend service for something this simple